execute/internal/gas/evm/gas_helpers.go

```diff
@@ -30,7 +37,38 @@ const (
 		100 //# SLOAD_GAS = WARM_STORAGE_READ_COST for rewriting from non-zero (in-progress) to non-zero (success/failure)
 )
 
+var (
+	abiUint32  = ABITypeOrPanic("uint32")
+	abiUint256 = ABITypeOrPanic("uint256")
+	abiBool    = ABITypeOrPanic("bool")
+
+	// ExtraArgsV2ABI ABI types for EVMExtraArgsV2 (see Client.sol)
+	ExtraArgsV2ABI = abi.Arguments{
```

Comment:
I guess referencing ABI/go-ethereum structs in the plugin code is forbidden as we want to stay chain agnostic. Some time ago I faced similar issues with abi encode/decode for the USDC specific attestation payloads. I was told to solve that for defining the interface in the `chailink-ccip` but providing implementation in the chainlink [here](https://github.com/smartcontractkit/chainlink/blob/66cd65e8e84848749853f907c700af154619cb2c/core/capabilities/ccip/ccipevm/tokendata.go)